### PR TITLE
Added dummyData file and added dummyData to state of App.jsx

### PR DESCRIPTION
**Pull request by rick-o-H**

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import RatingsAndReviews from './RatingsAndReviews/RatingsAndReviews';
+import { Products } from './dummyData.js';
+
+class App extends React.Component {
```
Comment (Gledinburgh) [inline client/src/App.jsx:6]: if you're importing { Component } on line 1, I don't think we need to  extend from React.Component, but I don't see the harm in having it.

Comment (rick-o-H) [reply client/src/App.jsx:6]: @Gledinburgh Yeah it auto-corrected to use React.component instead of Component for some reason lol

Comment (rick-o-H) [reply client/src/App.jsx:6]: @Gledinburgh Will change that!!

**Review by Gledinburgh — COMMENTED** [review]
Gorgeous work!

**Comment by rick-o-H** [comment]
@Gledinburgh Thanks, Glenn! I am not sure if my change went through.

**Pull request merged**